src/library_sigs.js

```diff
@@ -339,6 +339,8 @@ sigs = {
   _emval_await__sig: 'pp',
   _emval_call__sig: 'ppipp',
   _emval_call_method__sig: 'dppppp',
+  _emval_coro_make_promise__sig: 'ppp',
+  _emval_coro_suspend__sig: 'vpp',
```

Comment:
Wouldn't it be nicer to put the `__sig` alongside the source? Why do we have this separate sigs file?

Reply:
It's autogenerated by tooling from C/C++.